paladin-quests.lic

```diff
@@ -46,9 +47,20 @@ class PaladinQuests
     if args.warding
       warding
     elsif args.bonding
-      tamsine
+      case args.town
+      when 'shard'
+        args.town = 'Shard'
```

Comment:
In general I'm not a fan of mutating "input" (which I consider `args` to be). More a style thing than functional here, though.

Reply:
It's super weird that inputting Shard or shard will result in shard. Doesn't matter how the argument accepts it either. Do you know why this is and how to stop it from happening? It can't match shard to Shard in data-town, and I can't get .capitalize to work, maybe too old of a version?